src/components/Dialog.tsx

```diff
@@ -20,6 +20,13 @@ declare global {
   }
 }
 
+export enum Animation {
+  /** Used when the Dialog is closing. */
+  CLOSING = 'closing',
+  /** Used when the Dialog is paging to another Dialog screen. */
+  PAGING = 'paging',
```

Comment:
perhaps a good place to explain the concept of paging and why it's important

Reply:
done